sled-agent/src/instance_manager.rs

```diff
@@ -300,7 +300,7 @@ mod test {
             source_nat: SourceNatConfig {
                 ip: IpAddr::from(Ipv4Addr::new(10, 0, 0, 1)),
                 first_port: 0,
-                last_port: 1 << 14 - 1,
+                last_port: 1 << (14 - 1),
```

Comment:
This seems wrong to me.  I expect this was meant to be `(1 << 14) - 1`.  @bnaecker can you take a look?

Reply:
Yes that should be `16383 == (1 << 14) - 1`, nice catch Dave.